server/routes/index.js

```diff
@@ -60,10 +63,41 @@ router.get('/testLogin', isLoggedIn, function(req, res) {
     });
     
 /* GET home page. */
-router.get('/', function(req, res, next) {
+router.get('/', function(req, res, next) {7
   res.render('index', { pageTitle: 'Home' });
 });
 
+router.get('/map', function(req,res,next) {
```

Comment:
Not really a fan of this either, sort of jumping out of the realms of a plugin and into the framework of the underlying app. 
In saying that i don't know how to make it better either, @pagermon/collaborators any ideas?

Reply:
Ultimately was a quick put-together for the sake of making it work. Would appreciate input on how to improve the overall structure, and better ways to implement more advanced plugins in general.

Reply:
Ideally the route would be in the frontend js, and would pull data from API endpoint